src/AnypointTab.js

```diff
@@ -3,7 +3,7 @@ import '@polymer/paper-ripple/paper-ripple.js';
 import { AnypointButton } from '@anypoint-web-components/anypoint-button/src/AnypointButton.js';
 
 export class AnypointTab extends AnypointButton {
-  static get styles() {
+  get styles() {
     return [
       AnypointButton.styles,
```

Comment:
AnypointButton does not have static styles anymore. You changed it yourself :)
It should be `super.styles`